awswrangler/distributed/ray/modin/_data_types.py

```diff
@@ -18,7 +18,7 @@ def pyarrow_types_from_pandas_distributed(
     return ray_get(  # type: ignore
         func(
             df=first_block_object_ref,
-            index=index,
+            index=False,  # indices don't work with distributed Wrangler anyway
```

Comment:
This redshift unit [test](https://github.com/aws/aws-sdk-pandas/blob/release-3.0.0/tests/unit/test_redshift.py#L50) has started failing as a result of this change because `index` is set to True in the test and False is hardcoded here.

Reply:
The issue was that I realized `schema` wasn't getting passed to the distributed `read_*` methods. So in some cases the schema wasn't getting inferred properly. When I added it however, the fact that `schema` includes the index values started causing new problems. The distributed methods were failing to load indexes, so the Pandas -> PyArrow table conversion started failing because it couldn't find a column called `__index__0`. So I figured I should send a schema with everything other than indices.

I wasn't aware of the Redshift test failure though. I'll see what I can do there.